modules/administration_manual/pages/release_notes.adoc

```diff
@@ -17,6 +18,64 @@
 * xref:changes-in-8.0[Changes in 8.0]
 * xref:changes-in-7.0[Changes in 7.0]
 
+[[changes-in-10.1]]
+
+== Changes in 10.1
+
+Dear ownCloud administrator, please find below the changes and known issues in ownCloud Server 10.1 that need your attention. You can also read https://owncloud.org/changelog/server/[the full ownCloud Server changelog] for further details on what has changed.
+
+=== Semantic Versioning
+
+Starting with this release, ownCloud Server and the app ecosystem will follow the principles of https://semver.org/[Semantic Versioning].
+This step was taken to benefit operators by clearly indicating the contents and upgrade procedures of new releases via version numbers. Practically, the versioning scheme will follow the "Major.Minor.Patch" (or "Breaking.Feature.Fix") format.
+App developers need to re-release their apps to make them compatible with the new version. For details, please refer to https://owncloud.org/news/switching-owncloud-to-semantic-versioning/[this blog post].
+
+=== Change Management for Server Updates
+
+`occ upgrade` pulls app updates from the ownCloud Marketplace to make sure that not only the Server itself but also the installed apps are kept up-to-date. In line with the new versioning principles `occ upgrade` as well as the https://marketplace.owncloud.com/apps/market[Market App] now make a difference between major and minor app updates. Practically, this means that during a minor Server upgrade only new minor app versions will be installed. This is to make sure that apps with breaking changes will not be automatically installed when upgrading the Server. The `--major` option for `occ upgrade` and `occ market:install` provides the means for administrators to force installing new major app versions. Additionally, the https://marketplace.owncloud.com/apps/market[Market App] now includes a version picker to enable administrators to choose which version of an app they want to install or upgrade to.
```

Comment:
Server doesn't need an initial capital in "not only the Server itself".

Reply:
I consider "Server" a proper name which is why I capitalize it. Makes sense or not? :)